kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/client/DefaultFileSystemClientSuite.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.client
+
+import io.delta.kernel.defaults.utils.DefaultKernelTestUtils.getTestResourceFilePath
+import io.delta.kernel.defaults.utils.TestUtils
+import org.apache.hadoop.conf.Configuration
+import org.scalatest.funsuite.AnyFunSuite
+
+import java.io.FileNotFoundException
+import scala.collection.mutable.ArrayBuffer
+
+class DefaultFileSystemClientSuite extends AnyFunSuite with TestUtils {
+
+  val fsClient = new DefaultFileSystemClient(new Configuration())
```

Comment:
```suggestion
  val fsClient = defaultTableClient.getFileSystemClient
```

Reply:
@vkorukanti thank you, about "remove the TestDefaultFileSystemClient.java" if we do that for both #2639 and #2640 then the [client](https://github.com/delta-io/delta/tree/master/kernel/kernel-defaults/src/test/java/io/delta/kernel/defaults/client) package will be empty, maybe it needs to be removed as well. Should we have another PR like "remove redundant tests"?

Reply:
Separate PR to remove the redundant tests sounds good.